src/sage/modular/modform/ring.py

```diff
@@ -292,14 +293,15 @@ def group(self):
 
     def gen(self, i):
         r"""
-        Return the `i`-th generator of ``self``.
+        Return the `i`-th generator of this ring.
 
         INPUT:
 
-        - ``i`` (Integer) -- correspond to the `i`-th modular form generating
-          the ring of modular forms.
+        - ``i`` (Integer) -- correspond to the `i`-th modular form
+          generating the ring of modular forms.
 
```

Comment:
There is no rule, but I would recommend
```
- ``i`` -- integer corresponding to the `i`-th modular form ...
```
Generally, a common general style is
```
- ``input`` -- type (default: value); description
```
where type can be mathematical. In this case, an integer.

Or
```
- ``input`` -- (default: value) description
```
if type is obvious by the description. I think we may omit semicolon in this case.